frontend/src/components/pages/CampOverview/CampersTable/CampersTableMenu.tsx

```diff
@@ -0,0 +1,31 @@
+import {
+  Menu,
+  MenuButton,
+  IconButton,
+  MenuList,
+  MenuItem,
+} from "@chakra-ui/react";
+import React from "react";
+import { FaEllipsisV } from "react-icons/fa";
+
+const CampersTableMenu = (): JSX.Element => {
+  return (
+    <Menu placement="bottom-end">
+      <MenuButton as={IconButton} icon={<FaEllipsisV />} variant="" />
+      <MenuList minW={0} width="164px" p={0}>
+        <MenuItem height="48px">View details</MenuItem>
+        <MenuItem height="48px">Move camper</MenuItem>
+        <MenuItem height="48px">Edit</MenuItem>
+        <MenuItem
+          height="48px"
+          textColor="secondary.critical.100"
+          onClick={() => console.log("hi")}
```

Comment:
Are you planning to handle the remove in this component? I'm thinking there should be a passed function as a parameter into this child component from the parent, which would handle the close modal + open confirmation

Reply:
Same thing, will need callbacks for the other 3 menu items too?

Reply:
add props to the table and a quick draft of the edit modal to experiment with how the callbacks would trigger the modal